tests/test_definitions/column_aggregate_expectations/expect_column_stdev_to_be_between.json

```diff
@@ -1,161 +1,212 @@
-        "in": {
-          "column": "dist1"
-        },
-        "out": {
-          "success": true,
-          "observed_value": 1.1547005383792517
-        }
+  "expectation_type": "expect_column_stdev_to_be_between",
+  "datasets": [
+    {
+      "data": {
+        "dist1": [
+          1,
+          1,
+          3
+        ],
+        "dist2": [
+          -1,
+          0,
+          1
+        ],
+        "zero": [
+          0,
+          0,
+          0
+        ],
+        "missing": [
+          -1,
+          1,
+          null
+        ]
       },
-      {
-        "title" : "positive_test__exact_min_and_max",
-        "exact_match_out" : false,
-        "in": {
-          "column": "dist1",
-          "min_value": 1.1547005383792517,
-          "max_value": 1.1547005383792517
+      "tests": [
+        {
+          "title": "basic_positive_test",
+          "include_in_gallery": true,
+          "exact_match_out": false,
+          "tolerance": 0.000000000001,
+          "in": {
+            "column": "dist1",
+            "min_value": 0.5,
+            "max_value": 1.5
+          },
+          "out": {
+            "success": true,
+            "observed_value": 1.1547005383792517
+          }
         },
-        "out": {
-          "success": true,
-          "observed_value": 1.1547005383792517
+        {
+          "title": "vacuously_true",
+          "exact_match_out": false,
+          "tolerance": 0.000000000001,
+          "in": {
+            "column": "dist1"
+          },
+          "out": {
+            "success": true,
+            "observed_value": 1.1547005383792517
+          }
         },
-        "only_for": ["pandas", "sqlite", "mysql"]
-      },
-      {
-        "title" : "positive_test__exact_min_and_max_sqlalchemy",
-        "exact_match_out" : false,
-        "in": {
-          "column": "dist1",
-          "min_value": 1.1547005383792515,
-          "max_value": 1.1547005383792515
+        {
+          "title": "positive_test__exact_min_and_max",
+          "exact_match_out": false,
+          "in": {
+            "column": "dist1",
+            "min_value": 1.1547005383792517,
+            "max_value": 1.1547005383792517
+          },
+          "out": {
+            "success": true,
+            "observed_value": 1.1547005383792517
+          },
+          "only_for": [
+            "pandas",
+            "sqlite",
+            "mysql"
+          ]
         },
-        "out": {
-          "success": true,
-          "observed_value": 1.1547005383792515
+        {
+          "title": "positive_test__exact_min_and_max_bigquery",
+          "exact_match_out": false,
+          "in": {
+            "column": "dist1",
+            "min_value": 1.1547005383792517,
+            "max_value": 1.1547005383792517
+          },
+          "out": {
+            "success": true,
+            "observed_value": 1.1547005383792517
+          },
+          "only_for": [
+            "bigquery_cfe"
+          ]
```

Comment:
This was the test that was added for `bigquery_cfe`. The rest of the changes in the file are from the `linter` reformatting the json.